omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js

```diff
@@ -17,7 +17,7 @@ function PatientSearchWidget(configuration){
         attributeHeaders = attributeHeaders.concat('<th>'+value+'</th>');
     });
     var tableId = 'patient-search-results-table';
-    var tableHtml = '<table class="table table-sm table-responsive-sm table-responsive-md table-responsive-lg table-responsive-xl}" id="'+tableId+'">'+
+    var tableHtml = '<table class="table table-sm" id="'+tableId+'">'+
```

Comment:
My #1 hack for making things responsive is to try deleting most of the code

Reply:
@brandones  hi brandon,well done on this,on your last screens short   ie (Find patient) i see the a drop down menu when the screen is full,i was thinking we could have a drop down menu on only minimizing the screen,what do you think?

Reply:
Hi Herbert, this PR doesn't relate to the nav bar.